corehq/apps/users/credentials_issuing.py

```diff
@@ -60,3 +63,57 @@ def get_app_ids_by_activity_level():
     for app in credential_apps:
         app_ids_by_level[app.activity_level].append(app.app_id)
     return app_ids_by_level
+
+
+def submit_new_credentials():
+    from corehq.apps.users.models import UserCredential
+
+    credentials_to_submit, credential_ids_to_update = get_credentials_to_submit()
```

Comment:
Would you mind explaining the difference between `credentials_to_submit` and `credential_ids_to_update`. Why is there a difference?

Reply:
Ok, looking at the task where `submit_new_credentials` is called I think I can see the difference. 

It looks like the way it's now is, we create the `UserCredential` records, then only kick off `submit_new_credentials` where we read the `UserCredential` records from the DB again, but this time sending it off to PersonalID to actually provision the credentials. Is this a right understanding?